daisy/workflow/workflow.go

```diff
@@ -245,6 +257,49 @@ func (w *Workflow) populateStep(step *Step) error {
 	return step.SubWorkflow.workflow.populate()
 }
 
+func (w *Workflow) populateMergeWorkflow(step *Step) error {
+	step.MergeWorkflow.workflow.GCSPath = w.GCSPath
+	step.MergeWorkflow.workflow.Name = step.name
+	step.MergeWorkflow.workflow.Project = w.Project
+	step.MergeWorkflow.workflow.Zone = w.Zone
+
+	for k, v := range step.MergeWorkflow.Vars {
+		step.MergeWorkflow.workflow.AddVar(k, v)
+	}
+	if err := step.MergeWorkflow.workflow.populateVars(); err != nil {
+		return err
+	}
+
+	var replacements []string
+	for k, v := range w.autovars {
```

Comment:
How should autovars resolve in this case? As in, do these included workflow pieces have their own autovars?